IbaTools/FastFabric/upgrade.exp

```diff
@@ -124,7 +124,7 @@ File: /usr/lib/opa/tools/upgrade.exp" case_setup case_cleanup {
 			# given driver, its possible that its not on the media
 			# so we can't expect it to be installed
 			# for now assume Stack or Enablement tools are always on media
-			set drivers {{((OFED OPA Stack)|(OFA OPA Stack)|(OFED IBACM)|(OFA IBACM)|(Cornelis HFI Components)|(OPA Tools)|(OFED IP over IB)|(OFA IP over IB)|(OPA FM))}}
+			set drivers {{((OFED OPA Stack)|(OFA OPA Stack)|(OFED IBACM)|(OFA IBACM)|(Intel HFI Components)|(OPA Tools)|(OFED IP over IB)|(OFA IP over IB)|(OPA FM))}}
```

Comment:
A rebranding change seems to be getting undone here.

Reply:
Correct, thanks